src/Phinx/Console/Command/AbstractCommand.php

```diff
@@ -44,7 +44,7 @@ abstract class AbstractCommand extends Command
     protected const DEFAULT_SEED_TEMPLATE = '/../../Seed/Seed.template.php.dist';
 
     /**
-     * @var \Phinx\Config\ConfigInterface
+     * @var \Phinx\Config\ConfigInterface|null
```

Comment:
Can you actually have a null config?

Reply:
As the config is initialized in its own function, yes. It's why this was necessary for a period of time:
```php
        /** @var \Phinx\Config\ConfigInterface|null $config */
        $config = $this->getConfig();
        if (!$config) {
            $this->loadConfig($input, $output);
        }
```

to make phpstan happy as `$this->getConfig()` was annotated as always returning `ConfigInterface`, but on first call to `bootstrap()` (which holds the above reference), it would be null.